.eslintignore

```diff
@@ -7,4 +7,7 @@ content/*
 public/*
 
 # Ignore examples
-examples/*
\ No newline at end of file
+examples/*
+
+# Ignore all javascript files in src folder for translation purposes
+src/components/*
```

Comment:
Why is this changed?

Reply:
My bad. I was experimenting with somethings, and I changed this `.eslintignore` .

I'll change it back to its default